sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/model_batch_deployment.py

```diff
@@ -72,16 +73,16 @@ def __init__(
         super().__init__(
             name=name,
             endpoint_name=endpoint_name,
-            properties=properties,
-            code_path=code_path,
-            scoring_script=scoring_script,
-            environment=environment,
-            model=model,
             description=description,
+            type="model",
```

Comment:
We might need a more verbose type descriptor here, otherwise it will be harder to find a type name for the normal `BatchDeployment` model.